lib/src/test/kotlin/graphql/nadel/validation/NadelFieldValidationTest.kt

```diff
@@ -33,7 +34,7 @@ class NadelFieldValidationTest : DescribeSpec({
             )
 
             val errors = validate(fixture)
-            assert(errors.map { it.message }.isEmpty())
+            assertTrue(errors.map { it.message }.isEmpty())
```

Comment:
For some reason if you run individual tests with `kotlin.assert` it doesn't fail.